probe/kubernetes/pod.go

```diff
@@ -94,8 +95,8 @@ func (p *pod) GetNode(probeID string) report.Node {
 		RestartCount:          strconv.FormatUint(uint64(p.RestartCount()), 10),
 	}
 
-	if p.VolumeClaimName() != "" {
-		latests[VolumeClaim] = p.VolumeClaimName()
+	if len(p.VolumeClaimNames()) > 0 {
+		latests[VolumeClaim] = strings.Join(p.VolumeClaimNames(), "~p$")
```

Comment:
`"~p$"` should be a constant.
And is that necessary and/or sufficient?  What are the rules on volume claim names?

Reply:
PVC name must consist of lower case alphanumeric characters, `-` or `.`, and must start and end with an alphanumeric character.
We can use a single space or any special character to join them.

Reply:
OK, can you please add a comment about the rules and put the separator in a `const`.
I would probably just use `,` as the separator, if that works, but don't really mind.

Reply:
Sure @bboreham, `,` will work.
I'll update the PR soon.